torch/_dynamo/variables/builder.py

```diff
@@ -1604,8 +1605,20 @@ def wrap_listlike(self, value: Union[tuple, list, odict_values, NamedTuple]):
                 is_tensor=False,
             )
             tensor_list_proxy.node.meta["grapharg"] = grapharg
+            result = list_variable
+
+        else:
+            output = [
+                LazyVariableTracker.create(
+                    item,
+                    source=GetItemSource(self.get_source(), i),
+                )
+                for i, item in enumerate(value)
+            ]
+            result = BaseListVariable.cls_for_instance(value)(
+                output, source=self.source
+            )
 
-        result = BaseListVariable.cls_for_instance(value)(output, source=self.source)
```

Comment:
Small change to avoid creating `BaseListVariable` and `LazyVariableTracker` for the list stealing case.